pages/residents/[id]/allocations/[allocationId]/allocateworker.tsx

```diff
@@ -0,0 +1,63 @@
+import Seo from 'components/Layout/Seo/Seo';
+import { useRouter } from 'next/router';
+import PersonView from 'components/PersonView/PersonView';
+import { useAuth } from 'components/UserContext/UserContext';
+import { isBrowser } from 'utils/ssr';
+import { canUserAllocateWorkerToPerson } from 'lib/permissions';
+import { User } from 'types';
+import { useResident } from 'utils/api/residents';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import React from 'react';
+import AddWorkerAllocation from 'components/Allocations/AddWorkerAllocation/AddWorkerAllocation';
+
+const AddNewAllocationPage = (): React.ReactElement => {
```

Comment:
Maybe rename to `AddWorkerAllocationPage`? 

In this branch there are three pages with the name "AddNewAllocationPage" gets confusing when trying to debug.

Reply:
I'll change the name to "AllocateWorkerPage"